net/packet/ip6.go

```diff
@@ -16,13 +16,17 @@ type IP6 struct {
 	Hi, Lo uint64
 }
 
+// IP6FromRaw16 converts a raw 16-byte IPv6 address to an IP6.
+func IP6FromRaw16(ip [16]byte) IP6 {
```

Comment:
Actually, @josharian, I was wrong. Not three, but 4 IP types 😂 

@josharian is talking about deleting the `wgcfg` package and its type.

See also: https://github.com/inetaf/netaddr/issues/53